mender-auth/cli/actions.cpp

```diff
@@ -17,16 +17,23 @@
 #include <mender-auth/context.hpp>
 
 #include <common/conf.hpp>
+#include <common/events.hpp>
+#include <common/log.hpp>
 #include <common/path.hpp>
 
+#include <mender-auth/ipc/server.hpp>
+
 namespace mender {
 namespace auth {
 namespace cli {
 
 using namespace std;
 
+namespace events = mender::common::events;
+namespace mlog = mender::common::log;
```

Comment:
Nit: When not in the top-level namespace we can use `log` instead of `mlog` just fine

Just so it's consistent with the rest of the codebase